rules/manager.go

```diff
@@ -314,14 +325,36 @@ func (g *Group) run(ctx context.Context) {
 	tick := time.NewTicker(g.interval)
 	defer tick.Stop()
 
+	makeStale := func(s bool) {
+		if !s || g.opts.LookbackDelta < g.interval {
```

Comment:
Why is LookbackDelta being considered here? Whether we write stale markers should have nothing to do with how a subsequent query happens to be configured.

Reply:
it is a safety if someone deletes a rule with interval: 30m so that we don't have go routine out there for 1 hour.

Reply:
That's already how it works for scraping, let's keep things simple.

Reply:
I have removed that bit and simplified the tests.